src/generators/src/packagejson.js

```diff
@@ -1,9 +1,26 @@
 const { objToString } = require('../utils')
+const { exec } = require('child_process')
+
+function childProcess(process) {
```

Comment:
Why this function is necessary?

Reply:
From what I noticed in the reported bug (#88), in the project generation the dependencies weren't being installed. This function runs npm install, after creating the final package.json of the project.

Reply:
It seems if you put the dependency in `    let packages = [` var, should work, without need to run `npm install`, did you had tried this method?

Reply:
no but i will try

Reply:
putting dependency in package list:

![image](https://user-images.githubusercontent.com/657657/148834089-c53e1f18-8baf-467b-9414-99d1553cff65.png)

Reply:
Any other suggestions to fix this problem?

Reply:
@italojs could you please look at this?